packages/components/src/external-link/types.ts

```diff
@@ -12,4 +12,9 @@ export type ExternalLinkProps = {
 	 * The URL of the external resource.
 	 */
 	href: string;
+	/**
+	 * Allows for markup other than icons or shortcuts to be added to the menu item.
```

Comment:
This description should be adjusted, as this component isn't s 'menu item'.

Reply:
Sure, can you suggest a better description
Thanks